src/client/containers/BoardPageComponents/CandidateList/CandidateListPreDeadline.js

```diff
@@ -31,38 +34,41 @@ export default function CandidateListPreDeadline({
   useUpdateBallots(userId, boardId, candidates, draggedInit);
 
   const handleDelete = async (candidateId) => {
-    await deleteCandidate(userId, boardId, candidateId);
+    try {
+      const response = await deleteCandidate(userId, boardId, candidateId);
+      return response;
```

Comment:
You are not returning a response here. Deleting has no response, only errors that can be thrown.  
setBoardLoading(true); should not be called if an error is thrown right?

Reply:
I'm not able to try. I couldn't go to /api/documentation somehow. But I did the changes can you please have a look?